src/uwuw/uwuw_preprocessor.cpp

```diff
@@ -157,15 +160,16 @@ void uwuw_preprocessor::process_materials() {
 
     // check for missing materials
     if (found_grave == std::string::npos && found_vacuum == std::string::npos) {
-      if (material_library.count(mat_name) == 0) {
+      pyne::Material found_mat = material_library.get_material(mat_name);
```

Comment:
We should probably implement a `count` method or something equivalent on the PyNE side.  For now, can't we get `material_library.get_keys()` and check that?